app/scripts/blocker_focus.js

```diff
@@ -0,0 +1,25 @@
+console.log("#### AfD CONTENT-BLOCKER ####");
+
+import { Blocker } from "./blocker_general";
+
+let blocker = new Blocker([
+    {
+        selector: 'div.teaser',
+        type: 'big'
+    },
+    {
+        selector: 'div.promo',
+        type: 'big'
+    },
+    {
+        selector: 'li.teaser',
+        type: 'big'
+    },
+    {
+        selector: 'div.img',
+        type: 'big'
+    },
+]);
```

Comment:
I found some missing selectors: `div#article` for articles; `div.popularArticles` for "Meistgelesen". And did you check the search results as well?